common/config/rush/npm-shrinkwrap.json

```diff
@@ -4,18 +4,13 @@
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
-    "@hypnosphi/fuse.js": {
-      "version": "3.0.9",
-      "resolved": "https://registry.npmjs.org/@hypnosphi/fuse.js/-/fuse.js-3.0.9.tgz",
-      "integrity": "sha1-6pn2EhtKjwZbTHH4VZXbJxRJiAc="
-    },
     "@microsoft/api-extractor": {
-      "version": "4.3.4",
-      "resolved": "https://registry.npmjs.org/@microsoft/api-extractor/-/api-extractor-4.3.4.tgz",
-      "integrity": "sha1-YLgRx/KPIVPpM2qS26p06RWiA/Q=",
+      "version": "4.3.7",
+      "resolved": "https://onedrive.pkgs.visualstudio.com/_packaging/odsp-npm/npm/registry/@microsoft/api-extractor/-/api-extractor-4.3.7.tgz",
```

Comment:
I'm not sure why the "resolved" base urls all changed. I ran the command from rush v.4.1.0 so I'm not sure what other factors on my end could contribute to this change.

Reply:
This might be due to your user's.npmrc config--did you modify that recently (e.g. for doing product work)?

Reply:
ahhhh that could be! I did modify that recently for prod work. I'll take a look.

Reply:
Yep, that was the culprit!